Client/Extensions/UIImageViewExtensions.swift

```diff
@@ -26,6 +26,19 @@ extension UIImageView {
     }
     
     private func setIconURL(_ iconURL: String?, forURL url: URL?, completed completionBlock: ((UIColor, URL?) -> Void)?, scaledDefaultIconSize: CGSize? = nil) {
+        
+        // Priority order for favicons:
```

Comment:
Looking through the code, I believe this is true, but want to confirm.

This priority is just adding 1, 2/3 were already implemented this way correct? As in, it looks like you just cleaned up the code a bit but still functionally equivalence (minus the new #`)?

Reply:
Correct, added 1 new method, 2nd and 3rd without changes, just cleaned up this code slightly